source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/patterns/patterns_cfg.py

```diff
@@ -81,14 +86,13 @@ class PinholeCameraPatternCfg(PatternBaseCfg):
     """
 
     horizontal_aperture: float = 20.955
-    """Horizontal aperture (in mm). Defaults to 20.955mm.
+    """Horizontal aperture (in cm). Defaults to 20.955cm.
 
     Emulates sensor/film width on a camera.
 
     Note:
         The default value is the horizontal aperture of a 35 mm spherical projector.
```

Comment:
This should also be fixed?

Reply:
I am not sure bc I guess the 20.955 is still originating from a 35mm projector